.github/workflows/test-all.yml

```diff
@@ -7,18 +7,66 @@ on:
 env:
   # make chromedriver detect installed Chrome version and download the corresponding driver
   DETECT_CHROMEDRIVER_VERSION: true
+  artifactRetentionDays: 2
 
 jobs:
+  build:
+    name: Build the SDK
+    runs-on: ubuntu-latest
+    steps:
+    # Install Chrome so the correct version of webdriver can be installed by chromedriver when
+    # setting up the repo. This must be done to build and execute Auth properly.
+    - name: install Chrome stable
+      # Install Chrome version 110.0.5481.177-1 as some Auth tests start to fail on version 111.
+      # Temporary: Auth team will explore what's going wrong with the auth tests.
+      run: |
+        sudo apt-get update
```

Comment:
I don't think the special version of Chrome is needed for the build, just the auth test. I don't think Chrome is used during the build process. Let me know if that's wrong.

Reply:
It seems that the [auth tests fail 100% ](https://github.com/firebase/firebase-js-sdk/actions/runs/5267375408/jobs/9524427438) of the time if we mismatch these versions, unfortunately. :(